### PR TITLE
Updated repo topics

### PR DESCRIPTION
**Pull request by jerae-duffin**
Updated repo topics

### CODE REVIEW

File: docker-hub/repos/configure/index.md [hunk 1]
```diff
@@ -3,55 +3,47 @@ description: Configuring repositories on Docker Hub
...
-You get one private repository for free with your Docker Hub user account (not
-usable for organizations you're a member of). If you need more private
-repositories for your user account, upgrade your Docker Hub plan from your
-[Billing Information](https://hub.docker.com/billing/plan){: target="_blank" rel="noopener" class="_"} page.
+You get one free private repository with your Docker Hub user account (not
+available for organizations you're a member of). If you need more private
+repositories for your user account, upgrade your Docker Hub subscription from your [Billing Information](https://hub.docker.com/billing/plan){: target="_blank" rel="noopener" class="_"} page.
 
-Once you create the private repository, you can `push` and `pull` images to and
+Once you've create a private repository, you can `push` and `pull` images to and
```
Comment (craig-osterhout) [inline docker-hub/repos/configure/index.md:20]:
```suggestion
Once you've created a private repository, you can `push` and `pull` images to and
```

File: docker-hub/repos/configure/index.md [hunk 2]
```diff
@@ -3,55 +3,47 @@ description: Configuring repositories on Docker Hub
...
 
-- `Read-only` access allows users to view, search, and pull a private repository in the same way as they can a public repository.
-- `Read & Write` access allows users to pull, push, and view a repository Docker
-  Hub. In addition, it allows users to view, cancel, retry or trigger builds
-- `Admin` access allows users to Pull, push, view, edit, and delete a
-  repository; edit build settings; update the repository description modify the
-  repositories "Description", "Collaborators" rights, "Public/Private"
-  visibility, and "Delete".
+- `Read-only` access lets users view, search, and pull a private repository in the same way as they can a public repository.
+- `Read & Write` access lets users pull, push, and view a repository Docker
```
Comment (craig-osterhout) [inline docker-hub/repos/configure/index.md:39]: I just noticed this, but it wasn't caused by your changes.
Not sure if add `in` (`in Docker Hub`) or remove `Docker Hub`. Removing is more consistent with the other bullet points.
```suggestion
- `Read & Write` access lets users pull, push, and view a repository
```

File: docker-hub/repos/configure/index.md [hunk 3]
```diff
@@ -3,55 +3,47 @@ description: Configuring repositories on Docker Hub
...
+  Hub. In addition, it lets users view, cancel, retry or trigger builds
```
Comment (craig-osterhout) [inline docker-hub/repos/configure/index.md:40]: Continued from the previous line if removing `Docker Hub`.

```suggestion
  . In addition, it lets users view, cancel, retry, or trigger builds.
```

File: docker-hub/repos/index.md [hunk 4]
```diff
@@ -72,7 +78,7 @@ To avoid losing your private images, you can pull your private images from your
 
 1. Navigate to [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"} and select **Organizations**.
 2. Select the applicable organization and verify that your user account is a member of the organization.
-3. Sign in to [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"} using your original Docker ID, and pull your images from the initial namespace.
+3. Sign into [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"} using your original Docker ID, and pull your images from the initial namespace.
```
Comment (craig-osterhout) [inline docker-hub/repos/index.md:81]: Should this stay as `Sign in to`? This always trips me up, so I'm not sure.

File: docker-hub/repos/configure/index.md (hunk 1 above)
Comment (jerae-duffin) [reply docker-hub/repos/configure/index.md:20]: Thank you! Updated

File: docker-hub/repos/configure/index.md (hunk 2 above)
Comment (jerae-duffin) [reply docker-hub/repos/configure/index.md:39]: Thanks Craig! I removed it

File: docker-hub/repos/configure/index.md (hunk 3 above)
Comment (jerae-duffin) [reply docker-hub/repos/configure/index.md:40]: Updated

File: docker-hub/repos/index.md (hunk 4 above)
Comment (jerae-duffin) [reply docker-hub/repos/index.md:81]: Same =) Thank you! Updated for sign in to

**Review by craig-osterhout — APPROVED** [review]
LGTM

**Pull request merged**